server/src/controllers/board.controller.ts

```diff
@@ -50,14 +52,40 @@ class BoardController {
 		const boardRepository = getCustomRepository(BoardRepository);
 		const { id } = req.params;
 		try {
-			const projects=await boardRepository.getProjects(id);
+			const projects = await boardRepository.getProjects(id);
 
 			res.status(200).send(projects);
 		} catch (e) {
 			next(new ErrorResponse(HttpStatusCode.NOT_FOUND, e.message));
 		}
 	};
 
+	getBoardSprints = async (req: Request, res: Response, next: NextFunction) => {
+		const boardRepository = getCustomRepository(BoardRepository);
+		const sprintRepository = getCustomRepository(SprintRepository);
+
+		try {
+			const { id } = req.params;
+			const boardSprintsIds = (await boardRepository.getSprints(id)) as Sprint[];
+
+			const sprints = await Promise.all(
+				boardSprintsIds.map(async (sprintId) => {
+					let sprintData;
+
+					if (typeof sprintId === 'string') {
+						sprintData = await sprintRepository.findOneById(sprintId);
+					}
+
+					return sprintData;
```

Comment:
чому зразу не витягувати всі спринти за допомогою query builder?

Reply:
Using here sprintRepository.findOneById(sprintId) helps me to get detailed sprint data since I am loading eager relations too when I use findOneById() method.

```
async findOneById(id: string): Promise<Sprint> {
		return this.findOneOrFail(id, { loadEagerRelations: true });
	}
```

Reply:
Fixed in - 77875d2